app/instance/foleja/config/appConfig.ts

```diff
@@ -0,0 +1,261 @@
+    { name: 'Cognitive', id: 'cognitive', image: 'ic_act_cognitive' },
+    { name: 'Motor', id: 'motor', image: 'ic_act_movement' },
+  ],
+  activityCategoryobj: [
+    { name: 'Socio-emotional', id: 6431, image: 'ic_act_emotional' },
+    { name: 'Language and communication', id: 6441, image: 'ic_act_language' },
+    { name: 'Cognitive', id: 6436, image: 'ic_act_cognitive' },
+    { name: 'Motor', id: 6421, image: 'ic_act_movement' },
+  ],
+  articleCategoryobj: [
+    { name: 'playingAndLearning', id: 55, image: 'ic_artl_play' },
+    { name: 'healthAndWellbeingid', id: 2, image: 'ic_artl_health' },
+    { name: 'safetyAndProtection', id: 3, image: 'ic_artl_safety' },
+    { name: 'responsiveParenting', id: 56, image: 'ic_artl_responsive' },
+    { name: 'parentingCorner', id: 4, image: 'ic_artl_parenting' },
+    { name: 'nutritionAndBreastfeeding', id: 1, image: 'ic_artl_nutrition' },
+  ],
+  bothParentGender: 60,
+  bothChildGender: 59,
+  girlChildGender: 41,
+  boyChildGender: 40,
+  weightForHeight: 32786,
+  heightForAge: 6461,
+  today: new Date(),
+  restOfTheWorldCountryId: 126,
+  videoArticleMandatory: 0,
+  maxArticleSize: 5,
+  flavorName: 'Foleja',
+  apiConfig: {
+    articles: 'articles',
+    videoArticles: 'video-articles',
+    dailyMessages: 'daily-homescreen-messages',
+    basicPages: 'basic-pages',
+    sponsors: 'sponsors',
+    taxonomies: 'taxonomies',
+    standardDeviation: 'standard_deviation',
+    milestones: 'milestones',
+    activities: 'activities',
+    surveys: 'surveys',
+    childDevelopmentData: 'child-development-data',
+    childGrowthData: 'child-growth-data',
+    vaccinations: 'vaccinations',
+    healthCheckupData: 'health-checkup-data',
+    pinnedContent: 'pinned-contents',
+    checkUpdate: 'check-update',
+    faqs: 'faqs',
+    archive: 'archive',
+    countryGroups: 'country-groups',
+  },
+  measurementPlaces: (items: any) => [
+    { id: 0, title: items[0] },
+    { id: 1, title: items[1] },
+  ],
+  maxCharForRemarks: 200,
+  threeeMonthDays: 90,
+  twoMonthDays: 60,
+  oneMonthDays: 30,
+  afterDays: 5,
+  beforeDays: 6,
+  maxPeriodDays: 2920,
+  maxWeight: 33,
+  maxHeight: 125,
+  finalUrl: (
+    apiEndpoint: string,
+    selectedCountry: number | undefined,
+    selectedLang: string
+  ): string => {
+    const baseUrl = `${apiUrlDevelop}/${apiEndpoint}`;
+
+    switch (apiEndpoint) {
+      case config.apiConfig.countryGroups:
+        return `${baseUrl}/${config.flavorName}`;
+      case config.apiConfig.sponsors:
+        return `${baseUrl}/${selectedCountry}`;
+      case config.apiConfig.taxonomies:
+        return `${baseUrl}/${selectedLang}/all`;
+      case config.apiConfig.checkUpdate:
+        return `${baseUrl}/${selectedCountry}`;
+      default:
+        return `${baseUrl}/${selectedLang}`;
+    }
+  },
+  allApisObject: (isDatetimeReq: any, dateTimeObj: any): any => {
+    const allApiObject = [
```

Comment:
You can simplify by refactoring. Something like:

`const buildApiObject = (endpoint, saveinDB = true, isDatetimeReq = false, datetimeKey = '') => ({
  apiEndpoint: endpoint,
  method: 'get',
  postdata: isDatetimeReq && datetimeKey ? { datetime: datetimeKey } : {},
  saveinDB: saveinDB,
});`